streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/bean/AppBuildDockerResolvedDetail.java

```diff
@@ -43,7 +44,7 @@ public class AppBuildDockerResolvedDetail {
 
   public static AppBuildDockerResolvedDetail of(@Nullable DockerResolvedSnapshot snapshot) {
     AppBuildDockerResolvedDetail detail = new AppBuildDockerResolvedDetail();
-    if (snapshot == null) {
+    if (Objects.isNull(snapshot)) {
```

Comment:
If there is a lambda expression in this code, it will be more concise to use Objects.isNull, e.g:
```
List<String> list = Arrays.asList("a", "b", null, null);
        List<String> list2 = list.stream()
                .filter(Objects::nonNull)
                .collect(Collectors.toList())
```
.Conversely, it is not recommended to use Objects.isNull; it is better to directly check if the object is null, such as if (obj == null).

Reply:
Ok, thanks for reviewing the code